grub2/ignition.cfg

```diff
@@ -0,0 +1,15 @@
+# Determine if this is a first boot and set the ${ignition_firstboot} variable
```

Comment:
Might be worth adding some context here at this top of this file about how it's currently used.